src/Synthesizer.sol

```diff
@@ -70,7 +70,7 @@ contract Synthesizer is UUPSUpgradeable, OwnableUpgradeable {
         if (ipnft.ownerOf(ipnftId) != _msgSender()) {
             revert MustOwnIpnft();
         }
-        string memory tokenSymbol = string.concat(ipnft.symbol(ipnftId), "-MOL");
+        string memory tokenSymbol = ipnft.symbol(ipnftId);
```

Comment:
Why did you choose to use the ipnft symbol instead of removing it alltogether and just define a symbol during synthesis?

Reply:
It causes a higher gas usage. It’s bad but it’s wanted.